suite-native/module-accounts-import/src/screens/AssetsImportScreen.tsx

```diff
@@ -0,0 +1,147 @@
+    flex: 1,
+    justifyContent: 'space-between',
+}));
+
+const importAnotherWrapperStyle = prepareNativeStyle(_ => ({
+    alignItems: 'center',
+}));
+
+const importAnotherButtonStyle = prepareNativeStyle(utils => ({
+    borderRadius: utils.borders.radii.round,
+    width: 165,
+}));
+
+export const AssetsImportScreen = ({
+    navigation,
+    route,
+}: StackToTabCompositeScreenProps<
+    AccountsImportStackParamList,
+    AccountsImportStackRoutes.AccountImport,
+    RootStackParamList
+>) => {
+    const dispatch = useDispatch();
+    const [accountInfo, setAccountInfo] = useState<AccountInfo | null>(null);
+    const [selectedDevice, setSelectedDevice] = useState<DummyDevice>();
+    const [assetName, setAssetName] = useState<string>('bitcoines #1');
+
+    const { applyStyle } = useNativeStyles();
+
+    const { xpubAddress, currencySymbol } = route.params;
+
+    const getAccountInfo = useCallback(() => {
+        const showAccountInfoAlert = ({ title, message }: { title: string; message: string }) => {
+            Alert.alert(title, message, [
+                { text: 'OK, I will fix it', onPress: () => navigation.goBack() },
+            ]);
+        };
+
+        TrezorConnect.getAccountInfo({
+            coin: currencySymbol,
+            descriptor: xpubAddress,
+            details: 'txs',
+        })
+            .then(accountInfo => {
+                if (accountInfo?.success) {
+                    setAccountInfo(accountInfo.payload);
+                } else {
+                    showAccountInfoAlert({
+                        title: 'Account info failed',
+                        message: accountInfo.payload?.error ?? '',
+                    });
+                }
+            })
+            .catch(error => {
+                showAccountInfoAlert({
+                    title: 'Account info failed',
+                    message: error?.message ?? '',
+                });
+            });
+    }, [xpubAddress, currencySymbol, navigation]);
+
+    useEffect(() => {
+        getAccountInfo();
+    }, [getAccountInfo]);
+
+    const handleConfirmAssets = () => {
+        if (accountInfo && selectedDevice) {
+            const { title, value } = selectedDevice;
+            dispatch(
+                importAccountThunk({
+                    deviceId: value,
+                    deviceTitle: title,
+                    accountInfo,
+                    coin: currencySymbol,
+                }),
+            );
+            dispatch(setOnboardingFinished(true));
+            navigation.navigate(RootStackRoutes.App, {
+                screen: AppTabsRoutes.HomeStack,
+                params: {
+                    screen: HomeStackRoutes.Home,
+                },
+            });
+        }
+    };
+
+    const handleSelectDevice = (device: DummyDevice) => {
+        setSelectedDevice(device);
+    };
+
+    return (
+        <Screen>
+            {!accountInfo ? (
+                <AssetsLoader />
```

Comment:
I think we agreed to remove Loader at all in favor of some different solution like disabled/removed button etc.

Reply:
I know, but I would prefer to keep a loader without a forced timeout for now as it is present in wireframes and I have remembered I talked with Simon a few weeks before that there is some animation planned while getting account info.